library/edihistory/test_edih_sftp_files.php

```diff
@@ -378,3 +378,4 @@ function sftp_status($msg, $val)
     fwrite(($exitcd ? STDERR : STDOUT), xl($exitmsgs[$exitcd]).PHP_EOL);
     exit($exitcd);
 }
+}
```

Comment:
It looks like this bracket should instead be at line 188 where it seems like the closing bracket for that function there is missing.